sdk/storage/Azure.Storage.Internal.Avro/tests/AvroReaderTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.Storage.Internal.Avro.Tests
+{
+    public class AvroReaderTests
```

Comment:
I'd love to see a test case where we're skipping some records, so that we leverage option to pass two streams into it.